pyvistaqt/editor.py

```diff
@@ -17,6 +16,10 @@
     QVBoxLayout,
     QWidget,
 )
+try:  # backwards compatibility with pyvista<=0.29.0
```

Comment:
Local testing suggests the first line works on 0.29.0 but not 0.28.1 from PyPi
```suggestion
try:  # backwards compatibility with pyvista<0.29.0
```